debezium-server-redis/src/main/java/io/debezium/server/redis/RedisStreamChangeConsumer.java

```diff
@@ -95,7 +95,8 @@ else if (MESSAGE_FORMAT_COMPACT.equals(messageFormat)) {
             };
         }
 
-        RedisConnection redisConnection = new RedisConnection(config.getAddress(), config.getUser(), config.getPassword(), config.getConnectionTimeout(),
+        RedisConnection redisConnection = new RedisConnection(config.getAddress(), config.getDbIndex(), config.getUser(), config.getPassword(),
```

Comment:
@mfortunat The method `getDbIndex` is missing from `RedisStreamChangeConsumerConfig`